L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -233,8 +248,15 @@ L1FPGATrackProducer::L1FPGATrackProducer(edm::ParameterSet const& iConfig)
     tableTREFile = iConfig.getParameter<edm::FileInPath>("tableTREFile");
   }
 
+  // book ED output token for clock and bit accurate tracks
```

Comment:
Move these produce/consume statements about 12 lines earlier, so they're next to the other ones. The comment should make clear these are Tracklet tracks, since this EDProducer can optionally be configured to write TTTracks after the old KF fit.

Reply:
done